glitter_news/admin.py

```diff
@@ -35,7 +35,9 @@ def get_fieldsets(self, request, obj=None):
 
         fieldsets = (
             ('Post', {
-                'fields': ('title', 'category', 'author', 'date', 'image', 'summary', 'tags')
+                'fields': (
+                    'title', 'category', 'is_sticky', 'author', 'date', 'image', 'summary', 'tags',
```

Comment:
I disagree as it's a pretty common and well used feature across all CMSs... but I'll move it. :woman_shrugging: